packages/web/components/Quest/QuestFilter.tsx

```diff
@@ -34,89 +39,212 @@ export const QuestFilter: React.FC<Props> = ({
   aggregates,
   queryVariables,
   setQueryVariable,
+  roleChoices,
 }) => {
   const { user } = useUser();
   const myId = user?.id;
 
+  const limitOptions = [
+    {
+      label: '10',
+      value: '10',
+    },
+    {
+      label: '20',
+      value: '20',
+    },
+    {
+      label: '50',
+      value: '50',
+    },
+  ];
```

Comment:
Note that I switched from strings to numbers.

```suggestion
  const limitOptions = [10, 20, 50].map((n) => ({ label: n, value: n }))
```

Reply:
i'll use your syntax, but keep the strings, ok?

Reply:
One place you're using the values is in a string interpolation which will coerce it to a string. In the other you're converting the string to a number.

Why do you want to keep it as a string?

Reply:
to not have to make 2 types 😅 
`type ValueType = { value: string; label: string };`

but i can make another one. makes more sense. :)

Reply:
actually the whole `MetaFilterSelectSearch` is set up by requiring strings as values. in the `MetaFilterSelectSearch the values are passed to `SelectSearch`, which also requires these values as strings. so instead of refactoring all of that, imo we can just leave it as it is?

Reply:
Can you do `type ValueType = { value: string | number; label: string | number }` or `type ValueType<T = string> = { value: T; label: T }`? The second will have a string type by default, but you can use `ValueType<number>` to override it when you need to.

Reply:
renamed `ValueType` type to `FilterString`, just for clarity

Reply:
As I mentioned [elsewhere](//github.com/MetaFam/TheGame/pull/1085#discussion_r798335659), the brokenness of `MetaFilterSelectSearch` isn't something you have to deal with for this PR.